packages/foreman/rubygem-puma/rubygem-puma.spec

```diff
@@ -23,6 +23,7 @@ Requires: %{?scl_prefix}rubygem(nio4r) < 3
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}ruby-devel >= 2.2
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: openssl-devel
```

Comment:
Could you move this below the `# end` line so it doesn't get cleared on the next update?

Reply:
Done.